### PR TITLE
OSDOCS-1880: Cloud/Cluster Infra 4.8 Release Notes

### PR DESCRIPTION
**Pull request by lbarbeevargas**
4.8 Cloud/Cluster Infra release notes epic: https://issues.redhat.com/browse/OSDOCS-1880

Preview: https://deploy-preview-33148--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-machine-api

### CODE REVIEW

**Comment by lbarbeevargas** [comment]
@elmiko Release note for vSphere machine autoscaler to/from zero is ready for review. Thank you!

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -235,6 +235,10 @@ no operators found with name <name>.<version> in channel <name> of package <name
 [id="ocp-4-8-machine-api"]
 === Machine API
 
+[id="ocp-4-8-vsphere-machine-autoscaler-to-from-zero"]
+==== Scaling machines running in vSphere to and from zero with the machine autoscaler
```
Comment (elmiko) [inline release_notes/ocp-4-8-release-notes.adoc:239]: minor nit
```suggestion
==== Scaling machines running in vSphere to and from zero with the cluster autoscaler
```

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -235,6 +235,10 @@ no operators found with name <name>.<version> in channel <name> of package <name
...
+
+When running machines in vSphere, you can now set the `minReplicas` value to `0` in the `MachineAutoscaler` resource definition. When this value is set to `0`, the machine autoscaler scales the machine set to and from zero depending on if the machines are in use. For more information, see the xref:../machine_management/applying-autoscaling.html#machine-autoscaler-cr_applying-autoscaling[MachineAutoscaler resource definition].
```
Comment (elmiko) [inline release_notes/ocp-4-8-release-notes.adoc:241]: same here
```suggestion
When running machines in vSphere, you can now set the `minReplicas` value to `0` in the `MachineAutoscaler` resource definition. When this value is set to `0`, the cluster autoscaler scales the machine set to and from zero depending on if the machines are in use. For more information, see the xref:../machine_management/applying-autoscaling.html#machine-autoscaler-cr_applying-autoscaling[MachineAutoscaler resource definition].
```

**Review by elmiko — CHANGES_REQUESTED** [review]
looks mostly good, just a minor nit about the autoscaler name

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (lbarbeevargas) [reply release_notes/ocp-4-8-release-notes.adoc:239]: Good catch! Thank you 👍

**Comment by JoelSpeed** [comment]
With Mike's suggested fixes applied, LGTM

**Comment by lbarbeevargas** [comment]
@elmiko All set on updating "machine autoscaler" to "cluster autoscaler." 👍  Quick question on if I need to make the same wording update to the product docs pertaining to this feature as well:

> You can save on costs by setting this value to 0 for use cases such as running expensive or limited-usage hardware that is used for specialized workloads, or by scaling a machine set with extra large machines. The _machine autoscaler_ scales the machine set down to zero if the machines are not in use.

**Comment by lbarbeevargas** [comment]
@miyadav PTAL, thank you!

**Review by sagidlow — COMMENTED** [review]
Looks good! 
Could you also update the title of the PR:

> OSDOCS-1880: Cloud/Cluster Infra 4.8 Release Notes

Just so that we can comply with rules for labeling our PRs (:

**Comment by elmiko** [comment]
> Quick question: do I need to make the same wording update to the product docs pertaining to this feature as well?

good question. imo, in general we should be calling this thing the "cluster autoscaler". that is how the project is referred to in the community, it is also what that application does. so, yes i think we should be using that terminology.

imo, the confusion comes in here because we are manipulating API objects named `MachineAutoscalers`, and technically it is adding `Machines` to the cluster. i can understand how these terms could get jumbled up.

**Comment by listey** [comment]
Asked to look at this by @lbarbeevargas - all seems good to me.

**Pull request merged**